libs/wlgen/wlgen/rta.py

```diff
@@ -133,7 +133,8 @@ def calibrate(target):
                 pload[cpu] = int(pload_match.group(1))
                 log.debug('>>> cpu%d: %d', cpu, pload[cpu])
 
-        # target.cpufreq.load_governors()
+        for cpu, governor in old_governors.iteritems():
+            target.cpufreq.set_governor('cpu{}'.format(cpu), governor)
```

Comment:
I guess we should fix this in devlib then, i.e. make `get_all_governors` return a dict with `int`s for keys. I suggest submitting a PR and seeing what @setrofim says